packages/fast-tooling/src/web-components/css-layout/css-layout.template.ts

```diff
@@ -8,8 +8,8 @@ import { CSSLayout } from "./css-layout";
  */
 const flexDirectionRow = html`
     <svg
-        width="12"
-        height="12"
+        width="18"
```

Comment:
We want to avoid scaling icons because it puts lines and edges off-pixel. If we need larger icons we should adjust the svg. I also suggest we stick to more common sizes, like probably 16px live area here. I don't recall if these icons have built-in margins but I think not.

I can help with adjusting the sizes or finding the correct ones if they are already drawn.

Reply:
Responding in Teams.

Reply:
Reverted changes and replaced only the Wrap icon svgs.